docs/aqua_migration.rst

```diff
@@ -0,0 +1,13 @@
+#####################
+Qiskit Aqua Migration
```

Comment:
Also maybe we can make this "Qiskit Aqua Migration Guide" so links to it will say that

Reply:
Sure, changed.